tests/tasks/mixpanel/test_mixpanel_tasks.py

```diff
@@ -0,0 +1,168 @@
+        url = f"https://data.mixpanel.com/api/2.0/export?from_date=2011-07-10&to_date={today}"
+        responses.add(responses.GET, url, status=200, body="")
+
+        mx_export_task = MixpanelExportTask()
+        ret = mx_export_task.run(api_secret="foo")
+
+        assert ret is None
+
+    @responses.activate
+    def test_run_empty_result_with_parse(self):
+        today = date.today().strftime("%Y-%m-%d")
+        url = f"https://data.mixpanel.com/api/2.0/export?from_date=2011-07-10&to_date={today}"
+        responses.add(responses.GET, url, status=200, body="")
+
+        mx_export_task = MixpanelExportTask()
+        ret = mx_export_task.run(api_secret="foo", parse_response=True)
+
+        assert ret is None
+
+    @responses.activate
+    def test_run_without_parse(self):
+        today = date.today().strftime("%Y-%m-%d")
+        url = f"https://data.mixpanel.com/api/2.0/export?from_date=2011-07-10&to_date={today}"
+        responses.add(
+            responses.GET, url, status=200, body='{"foo": "bar"}\n{"alice": "bob"}'
+        )
+
+        mx_export_task = MixpanelExportTask()
+        ret = mx_export_task.run(api_secret="foo")
+
+        assert ret == '{"foo": "bar"}\n{"alice": "bob"}'
+
+    @responses.activate
+    def test_run_with_parse(self):
+        today = date.today().strftime("%Y-%m-%d")
+        url = f"https://data.mixpanel.com/api/2.0/export?from_date=2011-07-10&to_date={today}"
+        responses.add(
+            responses.GET, url, status=200, body='{"foo": "bar"}\n{"alice": "bob"}'
+        )
+
+        mx_export_task = MixpanelExportTask()
+        ret = mx_export_task.run(api_secret="foo", parse_response=True)
+
+        assert ret == [{"foo": "bar"}, {"alice": "bob"}]
+
+    @responses.activate
+    def test_run_with_parse_and_group(self):
+        today = date.today().strftime("%Y-%m-%d")
+        url = f"https://data.mixpanel.com/api/2.0/export?from_date=2011-07-10&to_date={today}"
+        event1 = '{"event": "alice", "properties": {"p1": "v1"}}'
+        event2 = '{"event": "bob", "properties": {"p1": "v1"}}'
+        event3 = '{"event": "bob", "properties": {"p2": "v2"}}'
+        body = "\n".join([event1, event2, event3])
+        responses.add(responses.GET, url, status=200, body=body)
+
+        mx_export_task = MixpanelExportTask()
+        ret = mx_export_task.run(
+            api_secret="foo", parse_response=True, group_events=True
+        )
+
+        assert ret == {"alice": [{"p1": "v1"}], "bob": [{"p1": "v1"}, {"p2": "v2"}]}
+
+    @responses.activate
+    def test_run_raises_fail(self):
+        url = "https://data.mixpanel.com/api/2.0/export?from_date=abc&to_date=abc"
+        responses.add(responses.GET, url, status=123, body="mixpanel error")
+
+        mx_export_task = MixpanelExportTask()
+
+        with pytest.raises(FAIL) as exc:
+            mx_export_task.run(api_secret="foo", from_date="abc", to_date="abc")
+
+        assert "Mixpanel export API error." in str(exc)
```

Comment:
You can use `match` to let pytest check the exception for you.
```suggestion
        with pytest.raises(FAIL, match="Mixpanel export API error.") as exc:
            mx_export_task.run(api_secret="foo", from_date="abc", to_date="abc")
```